ucte-network/src/test/java/eu/itesla_project/ucte/network/UcteFileReadWriteTest.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.ucte.network;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import eu.itesla_project.ucte.network.io.UcteReader;
+import eu.itesla_project.ucte.network.io.UcteWriter;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Iterator;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Christian Biasuzzi <christian.biasuzzi@techrain.it>
+ */
+public class UcteFileReadWriteTest {
+
+    private FileSystem fileSystem;
+
+    @Before
+    public void setUp() throws Exception {
+        fileSystem = Jimfs.newFileSystem(Configuration.unix());
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        fileSystem.close();
+    }
+
+    private void assertStreamsEquals(Stream<String> stream1, Stream<String> stream2) {
+        Iterator<String> iterS1 = stream1.iterator(), iterS2 = stream2.iterator();
+        while (iterS1.hasNext() && iterS2.hasNext()) {
+            assertEquals(iterS1.next().trim(), iterS2.next().trim());
+        }
+        assert (!iterS1.hasNext() && !iterS2.hasNext());
+    }
+
+    @Test
+    public void testExport() throws Exception {
+        Path sourceFile = new File(getClass().getResource("/20170322_1844_SN3_FR2.uct").toURI()).toPath();
+
+        UcteNetwork network = new UcteReader().read(Files.newBufferedReader(sourceFile));
```

Comment:
I'm not sure this is the proper way to get a Reader from a resource.
See new BufferedReader(new InputStreamReader(inputStream))

https://docs.oracle.com/javase/7/docs/api/java/io/InputStreamReader.html

Ignore this comment if you can inherit the ConverterBaseTest class